Templates/Ros2RoboticManipulationTemplate/README.md

```diff
@@ -0,0 +1,96 @@
+# ROS2  Robotic Manipulation Template
+
+This project is a template designed to help you quickly start simulating robotic arms in Open 3D Engine (O3DE) with Robot Operating System (ROS) 2.
+
+This template focuses on robotic arm manipulation with ROS 2 and [MoveIt 2](https://moveit.picknik.ai/main/index.html).
+
+
+## How to set up a new project with the template
+
+Please follow the instructions in [ROS 2 Gem documentation](https://development--o3deorg.netlify.app/docs/user-guide/interactivity/robotics/project-configuration/)
+to install all required dependencies and create your project with a template.
+
+If you followed the steps, your environment variables for `O3DE_HOME` and `PROJECT_PATH` will be set, your template registered.
+Create a new project with the following command:
+
+```shell
+${O3DE_HOME}/scripts/o3de.sh create-project --project-path $PROJECT_PATH --template-name Ros2RoboticManipulationTemplate
+```
+
+## Levels
+
+A new project created with this template has two levels to address two different groups of use-cases.
+
+### RoboticManipulation
+
+A level to try manipulation on, also suitable for solving picking and vision challenges in isolation. 
+It includes a [Panda Franka](https://www.franka.de/) manipulator, which is a popular choice in AI and manipulation research in the robotic community, RGBD camera, and some small objects to try manipulation with.
+
+The simulated robot is imported from public repository [panda_description](https://github.com/ros-planning/moveit_resources/tree/humble/panda_description).
+
+![Panda o3de](Screenshots/RoboticManipulation.png)
+
+### RoboticPalletization
+
+This level is more suitable for simulation of industrial use-cases with palletization. 
+Includes a warehouse scene, [UR10](https://www.universal-robots.com/products/ur10-robot/), which is an industrial, collaborative robot (cobot), and packages on a conveyor belt.
+
+![UR10 o3de](Screenshots/Palletization.png)
+
+## Examples 
+
+### Manipulation
+
+Install necessary ROS packages for MoveIt and depth image processing:
+
+```bash
+sudo apt-get install ros-humble-moveit ros-humble-moveit-resources ros-humble-depth-image-proc
```

Comment:
Nitpick: It is generally a good practice to keep the requirements in a different section.